types/jsdom/base.d.ts

```diff
@@ -39,16 +39,14 @@ declare module "jsdom" {
          *
          * @throws {Error} If the JSDOM was not created with `includeNodeLocations`
          */
-        nodeLocation(node: Node): ElementLocation | null;
+        nodeLocation(node: Node): Token.Location | null | undefined;
```

Comment:
Not `Token.ElementLocation`?

Reply:
see thread at https://github.com/DefinitelyTyped/DefinitelyTyped/pull/61171#discussion_r918888882

Reply:
Ah, nice!